linux/net.hhoney.candy.yml

```diff
@@ -0,0 +1,29 @@
+id: net.hhoney.candy
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+base: org.godotengine.godot.BaseApp
+base-version: '4.3-24.08'
+sdk: org.freedesktop.Sdk
+command: godot-runner
+finish-args:
+- --share=ipc
+- --socket=x11
+- --socket=pulseaudio
+- --device=all
+modules:
+- name: candy
+  buildsystem: simple
+  sources:
+
+  - type: dir
+    path: ../
+
+  - type: file
+    url: https://github.com/cassidyjames/CandyWrapper/releases/download/4.2/Candy.pck
```

Comment:
We can change this to a release artifact from this repo if you make a release from the existing 4.3 tag and upload your PCK file. Not super important, as this manifest is just for local testing and isn't the exact copy of what's submitted to Flathub, anyway.